cmd/pd-sidecar/main.go

```diff
@@ -28,11 +28,20 @@ import (
 	"github.com/llm-d/llm-d-inference-scheduler/pkg/sidecar/version"
 )
 
+var (
+	// supportedConnectors defines all valid P/D connector types
+	supportedConnectors = []string{
+		proxy.ConnectorNIXLV2,
+		proxy.ConnectorLMCache,
+		proxy.ConnectorSGLang,
+	}
+)
+
 func main() {
 	port := flag.String("port", "8000", "the port the sidecar is listening on")
 	vLLMPort := flag.String("vllm-port", "8001", "the port vLLM is listening on")
 	vLLMDataParallelSize := flag.Int("data-parallel-size", 1, "the vLLM DATA-PARALLEL-SIZE value")
-	connector := flag.String("connector", "nixlv2", "the P/D connector being used. Either nixl, nixlv2 or lmcache")
+	connector := flag.String("connector", "nixlv2", "the P/D connector being used. Either nixl, nixlv2, lmcache, or sglang")
```

Comment:
```suggestion
	connector := flag.String("connector", proxy.ConnectorNIXLV2, "the P/D connector being used. Supported: "+strings.Join(supportedConnectors, ", "))
```

It may be a good idea to update it this way.
And since `nixl` is no longer used, we should remove it from the list.

Reply:
That would be better. Thank you